src/main/java/org/opentripplanner/openstreetmap/tagmapping/FinlandMapper.java

```diff
@@ -84,7 +84,68 @@ else if (speedLimit <= 16.65f) {
 
     // No biking on designated footways/sidewalks
     props.setProperties("highway=footway", withModes(PEDESTRIAN));
-    props.setProperties("footway=sidewalk;highway=footway", withModes(PEDESTRIAN).walkSafety(1.0));
+    props.setProperties("footway=sidewalk;highway=footway", withModes(PEDESTRIAN));
+
+    // Walking on segregated ways is safer than when cycling and walking happens on the same lane
+    props.setProperties(
+      "highway=cycleway;segregated=yes",
+      withModes(PEDESTRIAN_AND_BICYCLE).walkSafety(1.1).bicycleSafety(0.6)
```

Comment:
Shouldn't `walkSafety` be below 1.0 in that case?

Reply:
The default walkSafety when walking on streets with bicycles is currently 1.15 https://github.com/opentripplanner/OpenTripPlanner/pull/4574/files#diff-e03d13c1794f61abb3292dc9370e1e0e3f38bb470c3f2b860cb74f7cde508e29R50